### PR TITLE
Setting up routes and server

### PR DESCRIPTION
**Pull request by jackzhen1996**

### CODE REVIEW

File: src/controllers/qaController.js [hunk 1]
```diff
@@ -1,21 +1,74 @@
 const db = require('../db');
 
+const loopAnswersToGetPhotos = (answers, object) => {
+  return Promise.all(answers.map(answer=>{
+    return db.query('SELECT * FROM photos WHERE answer_id = $1',[answer.id])
```
Comment (anna-zhu) [inline src/controllers/qaController.js:5]: What does $1 do? I'm not familiar with that syntax

File: src/controllers/qaController.js [hunk 2]
```diff
@@ -1,21 +1,74 @@
...
+};
+
+
+
 exports.getQuestions = (req,res) => {
   const responseObj = {};
   const {product_id, count} = req.query;
 
-  db.query(`SELECT * from questions WHERE product_id = $1 and reported = $2 limit $3`, [product_id, 0, count || 5])
+  db.query('SELECT * FROM questions WHERE product_id = $1 ORDER BY question_date DESC limit $2', [product_id,  count || 5])
```
Comment (anna-zhu) [inline src/controllers/qaController.js:61]: For Q&A, the only sorting that needs to happen is that the more helpful q/a appear first

**Review by anna-zhu — APPROVED** [review]
Looks good to me, nice work

File: src/controllers/qaController.js (hunk 1 above)
Comment (jackzhen1996) [reply src/controllers/qaController.js:5]: $1 is just a variable for the query, so for instance, db.query('select ... where id = $1 and id = $2', [a,b]), $1 has the value 'a' in the argument array, and $2 has the value of 'b' in the argument array

Comment (anna-zhu) [reply src/controllers/qaController.js:5]: Just got to the part about jsonpath variables LOL makes sense now

**Pull request merged**